### PR TITLE
Add Rails 5.1 support to JBuilder

### PR DESCRIPTION
**Pull request by maclover7**
Closes https://github.com/rails/jbuilder/issues/388
Closes https://github.com/rails/rails/issues/28140

### CODE REVIEW

File: gemfiles/rails_4_2.gemfile [hunk 1]
```diff
@@ -9,6 +9,5 @@ gem "pry"
 gem "railties", "~> 4.2.0"
 gem "actionpack", "~> 4.2.0"
 gem "activemodel", "~> 4.2.0"
-gem "nokogiri", "< 1.7" if RUBY_VERSION < "2.1"
```
Comment (rafaelfranca) [inline gemfiles/rails_4_2.gemfile]: Why this was removed?

Comment (maclover7) [inline gemfiles/rails_4_2.gemfile]: ... I didn't remove it? It might have gotten nixed when I ran `appraisal install`.

**Comment by maclover7** [comment]
@rafaelfranca should v2.6.3 be released with these changes?

**Comment by rafaelfranca** [comment]
Yes. I'm doing it already.

**Comment by rafaelfranca** [comment]
Done

**Comment by chrisarcand** [comment]
Thanks guys! 💙

**Pull request merged**